lib/exe/js.php

```diff
@@ -173,7 +173,7 @@ function js_load($file){
         // is it a include_once?
         if($match[1]){
             $base = utf8_basename($ifile);
-            if($loaded[$base]){
+            if(array_key_exists($base, $loaded)){
```

Comment:
This is not the same! The original checks if the value in the key evaluates to true. You're just checking if the key exists.